src/components/search/History/SearchHistory.tsx

```diff
@@ -36,7 +36,12 @@ const SearchHistory = ({ onClear }: SearchHistoryProps) => {
       <UIList variant="ordered">
         {searchHistory.map((item, index) => (
           <li data-store-search-history-item key={index}>
-            <Link variant="display" to={doSearch(item)} target="_blank">
+            <Link
+              variant="display"
+              to={doSearch(item)}
+              target="_blank"
+              rel="noreferrer"
```

Comment:
Is it intentional?

Reply:
_Kind of._ It was an auto fix done by the linter: 

> Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see [react/jsx-no-target-blank](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/jsx-no-target-blank.md)